modules/mariadb/src/test/java/org/testcontainers/containers/MariaDBR2DBCDatabaseContainerTest.java

```diff
@@ -20,4 +23,27 @@ protected String createR2DBCUrl() {
     protected MariaDBContainer<?> createContainer() {
         return new MariaDBContainer<>(DockerImageName.parse("mariadb:10.3.39"));
     }
+
+    @Test
+    public void testGetR2DBCUrl() {
+        //given
+        MariaDBContainer<?> container = createContainer();
+        container.start();
+
+        String expectedUrl =
+            "r2dbc:mariadb://" +
+            container.getHost() +
+            ":" +
+            container.getMappedPort(MariaDBContainer.MARIADB_PORT) +
+            "/" +
+            container.getDatabaseName() +
+            container.constructUrlParameters("?", "&");
+
+        //when
+        String r2dbcUrl = MariaDBR2DBCDatabaseContainer.getR2dbcUrl(container);
+        //then
+        assertThat(expectedUrl).isEqualTo(r2dbcUrl);
+        //clean-up
+        container.stop();
```

Comment:
avoid comments such as //given, //when, //then, //cleanup

I think the test such test connectivity, performing a query would be enough

Reply:
@eddumelendez 
I think you're right. I just removed the comments. Thanks for the code review!

Reply:
Can you update the test performing a query.

Also, use `try-with-resources` and remove `container.close()`.

Reply:
@eddumelendez 
Could you please check the code that I have updated to reflect the requested changes?